homeassistant/components/sensor/nzbget.py

```diff
@@ -173,8 +173,4 @@ def post(self, method, params=None):
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Update cached response."""
-        try:
-            self.status = self.post('status')['result']
-        except requests.exceptions.ConnectionError:
-            # failed to update status - exception already logged in self.post
-            raise
```

Comment:
We shouldn't let the exception be raised. We should return after catching it and logging it once.

Reply:
I agree, however, he merely rewrote the code to have same behavior as before.

Reply:
Yes, but it's now harder to spot. But since we've flagged it here. We should be able remember to fix it.